<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithm Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Properties">
      <attribute name="title">
       <string>Algorithm properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="l_NameAlgorithm">
         <property name="text">
          <string>Name of the algorithm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="le_NameAlgorithm"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="le_Version">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QTextEdit" name="te_Description"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="l_Version">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="l_Description">
         <property name="text">
          <string>Description of the algorithm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_Author">
         <property name="text">
          <string>Author of the algorithm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="le_Author"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Algorithm">
      <attribute name="title">
       <string>Algorithm steps</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="4">
        <widget class="QPushButton" name="b_AddBefore">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/add_above.png</normaloff>:/Buttons/icon/add_above.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="3">
        <widget class="QListView" name="lst_Functions"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="b_MoveDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/move_down.png</normaloff>:/Buttons/icon/move_down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="b_Delete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/bin.png</normaloff>:/Buttons/icon/bin.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="b_MoveUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/move_up.png</normaloff>:/Buttons/icon/move_up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="b_AddAfter">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/add_below.png</normaloff>:/Buttons/icon/add_below.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="b_Help">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Icons.qrc">
           <normaloff>:/Buttons/icon/help.png</normaloff>:/Buttons/icon/help.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="tv_algorithm"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
